authz-module/index.md

```diff
@@ -0,0 +1,204 @@
+:::
+
+Run the following commands to configure the `simd` binary.
+
+Set the chain ID:
+```
+simd config chain-id demo
+```
+
+Set the keyring backend:
+```
+simd config keyring-backend test
+```
+
+## Key Setup
+
+Create a couple test keys. You will use the `--recover` option so that the addresses used within the example commands below are consistent.
+
+Add a key for Alice:
+```
+simd keys add alice --recover
+```
+
+Enter the following mnemonic:
+```
+plunge hundred health electric victory foil marine elite shiver tonight away verify vacuum giant pencil ocean nest pledge okay endless try spirit special start
+```
+
+Add a key for Bob:
+```
+simd keys add bob --recover
+```
+
+Enter the following mnemonic:
+```
+shuffle oppose diagram wire rubber apart blame entire thought firm carry swim old head police panther lyrics road must silly sting dirt hard organ
+```
+
+## Chain Setup
+
+The following steps will start a chain for the simulation application (`simapp`).
+
+Initialize the node:
+```
+simd init test --chain-id demo
+```
+
+Add Alice and an initial balance to the genesis file:
+```
+simd add-genesis-account alice 5000000000stake
+```
+
+Add Bob and an initial balance to the genesis file:
+```
+simd add-genesis-account bob 5000000000stake
+```
+
+Generate a transaction adding Alice to the initial validator set:
+```
+simd gentx alice 1000000stake --chain-id demo
+```
+
+Add the validator transaction to the genesis file:
+```
+simd collect-gentxs
+```
+
+## Start Chain
+
+You are now ready to start a single node network on your local machine.
+
+Start the chain:
+```
+simd start
+```
+
+## Submit Proposal
+
+In order to demonstrate an authorization to vote on a governance proposal, you'll need to first create a governance proposal.
+
+Create proposal:
+```
+simd tx gov submit-proposal --title="Test Authorization" --description="Is Bob authorized to vote?" --type="Text" --deposit="10000000stake" --from alice
+```
+
+View proposal:
+```
+simd q gov proposal 1
+```
+
+## Grant Authorization
+
+Next, the granter will need to `grant` an authorization to the grantee.
+
+The authorization will be a "generic" authorization, meaning there are no limits on the authorization. In this case, the grantee will be able to vote as many times as they want until the granter revokes the authorization.
+
+Create authorization:
+```
+simd tx authz grant cosmos1khljzagdncfs03x5g6rf9qp5p93z9qgc3w5dwt generic --msg-type /cosmos.gov.v1beta1.MsgVote --from alice
+```
+
+View authorization:
+```
+simd q authz grants cosmos1jxd2uhx0j6e59306jq3jfqs7rhs7cnhvey4lqh cosmos1khljzagdncfs03x5g6rf9qp5p93z9qgc3w5dwt /cosmos.gov.v1beta1.MsgVote
+```
+
+## Generate Transaction
+
+In order for the grantee to execute a message on behalf of the granter, the grantee will first need to generate an unsigned transaction where the transaction author (the `--from` address) is the granter. 
```

Comment:
```suggestion
In order for the grantee to execute a message on behalf of the granter, the grantee must first generate an unsigned transaction where the transaction author (the `--from` address) is the granter. 
```

Reply:
in most cases, we can omit "In order for" and "In order to" and replace with "For" and "To" (just a nit for clarity)